.sass-lint.yml

```diff
@@ -2,57 +2,58 @@
 # https://github.com/sasstools/sass-lint/tree/master/docs/rules
 #
 files:
-  include: '**/*.s+(a|c)ss'
   ignore:
-    - 'vendor/**/*.scss'
-    - 'node_modules/**/*.scss'
+    - vendor/**/*.scss
+    - node_modules/**/*.scss
+    - kss/builder/decanter/**/*.scss
+  include: "**/*.s+(a|c)ss"
 rules:
-  # Class Name Format
+  brace-style:
+    - 1
+    -
+      style: stroustrup
   class-name-format:
     - 1
     -
       convention: hyphenatedbem
+  empty-line-between-blocks: 1
+  extends-before-declarations: 1
+  extends-before-mixins: 1
+  final-newline: 1
+  force-attribute-nesting: 0
+  force-pseudo-nesting: 0
   function-name-format:
     - 1
     -
-      convention: snakecase
-  # Consistency Rules
-  extends-before-mixins: 1
-  extends-before-declarations: 1
-  mixins-before-declarations: 1
-  # Require an empty line between blocks
-  empty-line-between-blocks: 1
-  no-empty-rulesets: 1
-  final-newline: 1
-  no-ids: 1
-  leading-zero: 1
+      convention: hyphenatedlowercase
+  hex-length: 0
+  leading-zero:
+    - 1
+    - include: true
+  mixins-before-declarations: 0
   name-format: 0
   nesting-depth:
     - 1
     -
       max-depth: 4
+  no-color-literals: 0
+  no-css-comments: 0
+  no-empty-rulesets: 1
+  no-ids: 1
+  no-mergeable-selectors: 0
+  no-misspelled-properties: 1
+  no-vendor-prefixes: 0
+  no-url-domains: 0
+  no-warn: 0
```

Comment:
```suggestion
  no-warn: 0
    no-qualifying-elements:
    - 1
    -
      allow-element-with-attribute: 1
```

Reply:
I also pushed up a small fix in CTA after the new linter found a small issue. Linter is working!